frontend/src/main/pages/AdminEditCommonsPage.js

```diff
@@ -1,75 +1,82 @@
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
 import { useParams } from "react-router-dom";
 import CommonsForm from "main/components/Commons/CommonsForm";
-import { Navigate } from 'react-router-dom'
+import { Navigate } from "react-router-dom";
 import { useBackend, useBackendMutation } from "main/utils/useBackend";
 import { toast } from "react-toastify";
 
 export default function CommonsEditPage() {
-  let { id } = useParams();
+    let { id } = useParams();
```

Comment:
Should this be indented back?

Reply:
This is just the prettier formatting, the only reason I use it is because its very commonly used in industry